x-pack/plugin/security/src/main/java/org/elasticsearch/xpack/security/rest/action/service/RestGetServiceAccountCredentialsAction.java

```diff
@@ -23,7 +21,6 @@
 
 import static org.elasticsearch.rest.RestRequest.Method.GET;
 
-@ServerlessScope(Scope.INTERNAL)
 public class RestGetServiceAccountCredentialsAction extends SecurityBaseRestHandler {
```

Comment:
I think we should keep this API internal instead of blocking it entirely. It is a useful debugging tool because it tells you what file-based service tokens are available on which node in addition to index-based service tokens. It helps to confirm file-based tokens are provisioned consistently for all nodes. The response does not involve any secret either.

Reply:
Thanks for the context. Agreed, it makes sense to keep it. I've pushed https://github.com/elastic/elasticsearch/pull/98492/commits/f1c783be6819e84b0c2b4af42684af0e3ec90bd5 to revert it back to internal.